components/Breadcrumb.vue

```diff
@@ -1,178 +1,161 @@
 <template>
```

Comment:
This is now _almost_ the same as the BCRS Shared Component version, but refactoring this to use the shared component is an exercise for the future (not in scope of this already-large ticket).